src/birthplace.js

```diff
@@ -128,10 +128,10 @@ function parseMalaysia(code) {
 }
 
 function parseForeign(code) {
-    return Object.assign(
-        { state: null },
-        countryCodePairs[code]
-    );
+    return {
```

Comment:
`Object.assign` is used to natively support the most amount of browsers. Spread operator wold get transpiled to `_objectSpread` to support even older browsers that are too old to support, and it adds to the bundle size (see browser bundle above)

Reply:
I notice the bundle size is about 1kb larger for this change, which was to follow section 3.8 of this [style guide](https://github.com/airbnb/javascript)

But, if bundle size matters, please ignore this change.